funding/manager.go

```diff
@@ -2313,7 +2310,7 @@ func (f *Manager) handleFundingCreated(peer lnpeer.Peer,
 	// With a permanent channel id established we can save the respective
 	// forwarding policy in the database. In the channel announcement phase
 	// this forwarding policy is retrieved and applied.
-	err = f.saveInitialFwdingPolicy(channelID, &forwardingPolicy)
+	err = f.saveInitialForwardingPolicy(channelID, &forwardingPolicy)
```

Comment:
I think the funder receives `accept_channel`, `funding_signed` and `channel_ready`, and the fudee receives `open_channel`, `funding_created`, and `channel_ready`.

This means `handleFundingAccept` and `handleFundingSigned` are both called by the funder, which means the policy is saved twice via `saveInitialForwardingPolicy`.

Reply:
I checked this again to make sure we're not actually saving things twice. There are 2 calls to `saveInitialForwardingPolicy`, one in `handleFundingSigned` (funder) and one in `handleFundingCreated` (fundee). And then a third one in `ensureInitialForwardingPolicy` which is called by `handleChannelReady` (funder and fundee) when making sure we have a forwarding policy before sending the channel to `brontide`.
So I think everything checks out.

Reply:
Correct, that is just for the cases where `lnd` is upgraded between opening a channel and it being confirmed.